AppCenterDistribute/AppCenterDistribute/MSACDistribute.m

```diff
@@ -1322,3 +1345,31 @@ + (void)resetSharedInstance {
 }
 
 @end
+
+@implementation MSACDistribute (ContextProviding)
+
+- (ASPresentationAnchor)presentationAnchorForWebAuthenticationSession:(ASWebAuthenticationSession *)session API_AVAILABLE(ios(13)) {
+  id<MSACDistributeDelegate> delegate = self.delegate;
+  if ([delegate respondsToSelector:@selector((distributeAuthenticationPresentationAnchor:))]) {
+    return [delegate distributeAuthenticationPresentationAnchor:self];
+  }
+
+  UIApplication *application = MSAC_DISPATCH_SELECTOR((UIApplication * (*)(id, SEL)), [UIApplication class], sharedApplication);
```

Comment:
I copied this whole bit from `MSACAlertController`. 
I'm guessing that it can be simplified for everything except accessing `sharedApplication`, but I'm unsure why the alert controller is using `MSAC_DISPATCH_SELECTOR` so heavily.

Reply:
It was there for keeping compatibility with older Xcode, that reqired for keeping bitcode compatibility.